src/app/app.component.spec.ts

```diff
@@ -13,17 +13,19 @@ import { TopNavigationSmartComponent } from '@shared/components/navigation/top-n
 !environment.testIntegration
   ? Logger.log('Integration skipped')
   : describe('[Integration] AppComponent', () => {
-      beforeEach(async(() => {
-        TestBed.configureTestingModule({
-          imports: [RouterTestingModule],
-          declarations: [
-            AppComponent,
-            MockComponent(FooterComponent),
-            MockComponent(SideNavigationSmartComponent),
-            MockComponent(TopNavigationSmartComponent),
-          ],
-        }).compileComponents();
-      }));
+      beforeEach(
+        waitForAsync(() => {
```

Comment:
Awesome! Nice to see we are updating to use the latest testing standards from Angular.